### DISCUSSION TITLE
Fan setting seems to follow the heater setting

### DISCUSSION THREAD

**Discussion by RickGroszkiewicz** [artisan-roaster-scope/artisan#745] [Q&A]
I recently set up artisan on a different laptop running Linux Mint 20. The software behaves differently on my Hottop B-2K+ than it did on my prior laptop. The fan setting seems to follow the heater setting, which requires me to manually correct the fan setting during each roast.

I recently did two roasts of the same bean. The roast on the old Hottop was what I expected. The roast on the new Hottop was "unusual":

<img width="868" alt="Sidamo WP 2019 Decaf-Roast 2" src="https://user-images.githubusercontent.com/25457554/144277660-c26673df-2c73-490b-ba06-e7bbe4318668.png">

**Comment by RickGroszkiewicz** [comment]
Here are the events recorded during the roast. The suspect ones are the "Q events" - which are not created by my alarm file.

<img width="866" alt="Sidamo WP 2019 Decaf-Roast 2-events" src="https://user-images.githubusercontent.com/25457554/144278166-5803074d-8aa2-4148-8489-022553579bc6.png">

**Comment by roasterdave** [comment]
Do you have Quantifiers enabled?  Config>> Events>> Quantifiers tab.

**Reply by RickGroszkiewicz** [reply]
Apparently they are enabled, but I didn't set this up (never used quantifiers). This must have been done automatically as part of the initial configuration via Config / Machine / Hottop / B2K+ 

<img width="864" alt="New Hottop-quantifiers" src="https://user-images.githubusercontent.com/25457554/144301800-181935d0-0a8e-431f-ae27-14efa33876eb.png">

**Comment by MAKOMO** [comment]
Could it be that your event types are wrongly labeled and thus the alarms trigger the wrong events. I see here one named "None".

M

**Comment by roasterdave** [comment]
Looks like the events were renamed?  The default order for the Hottop B-2K is Air, Drum, Damper Burner.

**Reply by RickGroszkiewicz** [reply]
Yes, I did have to rename the event types. This was required so I could use my existing alarm files that were created last year on a different laptop running Linux Mint 18:

![Config-Events-Config change event types for B-2K](https://user-images.githubusercontent.com/25457554/144344154-e3c7879a-2576-4621-a671-21698dbeea11.png)

**Reply by RickGroszkiewicz** [reply]
Here are the alarms that were triggered during the roast:

<img width="864" alt="Sidamo WP 2019 Decaf-Roast 2-alarms" src="https://user-images.githubusercontent.com/25457554/144347739-169b77ea-7754-44dd-b660-801947043ffc.png">

**Comment by RickGroszkiewicz** [comment]
After I changed the quantifier settings to match the event types, I got the expected results in a simple test roast. 
Here are the results for Config / Events / Quantifiers:
![config-events-quantifiers-2](https://user-images.githubusercontent.com/25457554/145650347-d9029c8f-7f20-401c-9eb9-98810826da67.png)

Here are the events that occurred during the test roast:
![roast events-after change quantifiers-2](https://user-images.githubusercontent.com/25457554/145650360-7671c0c8-57fb-44b9-a926-5f5f77f8cb14.png)

**Comment by roasterdave** [comment]
Good to hear!  Be sure to save your settings to a file (Help>> Save settings).

**Reply by RickGroszkiewicz** [reply]
Unfortunately, I have more research to do on this problem. I basically started over from scratch with the test roast, as shown in my prior post.

During my recent roasting session, the "Q events" still occur. When I set the air to 25% via my alarm file, a "Q event" reduces it to 20%. Then I had to manually change the air back to 25%.:

![yemen roast-Q events-air 20 versus 25](https://user-images.githubusercontent.com/25457554/145690622-607f8faf-604a-424f-a827-3acacb16c034.png)

I missed the change to 20% air near the end of the roast, which had little effect anyway.

![yemen roast-alarms](https://user-images.githubusercontent.com/25457554/145690631-480c9038-a6b1-4ee5-857d-d44d90972254.png)

**Reply by RickGroszkiewicz** [reply]
This looks like the effect of a "coarse" setting for the air. I carefully checked my settings (and looked in the file with a text editor), but none of the check boxes for "coarse" are selected.
